content/en/Getting started/What to Expect/report-contents.md

```diff
@@ -0,0 +1,162 @@
+---
+title: "What's in a Pentest Report"
+linkTitle: "Contents of a Pentest Report"
+weight: 310
+description: >
+  Here's what can you expect in a Pentest Report.
+---
+
+{{% pageinfo %}}
+Our pentest reports include what you need to further secure your systems.
```

Comment:
Do our pentesters really include "all the information" that our customers need? (Do we include that commitment in our SLA?) I hesitate to make an "absolutist" statement.

Also, "modern day cyberthreats" sounds like marketing, which I want to avoid in standard technical documentation